dali/operators/image/convolution/gaussian_blur.cc

```diff
@@ -34,36 +34,38 @@ constexpr static const char* kSigmaArgName = "sigma";
 constexpr static const char* kWindowSizeArgName = "window_size";
 
 DALI_SCHEMA(GaussianBlur)
-    .DocStr(R"code(Apply Gaussian Blur to the input.
+    .DocStr(R"code(Applies the Gaussian Blur to the input.
 
-User can specify sigma, kernel window size or both.
-If only the sigma is provided, the radius of kernel is calculated as ``ceil(3 * sigma)``,
-thus the kernel window size is ``2 * ceil(3 * sigma) + 1``.
+You can specify the sigma window size, the kernel window size, or both.
+If only the sigma window size is specified, the radius of kernel is calculated as
+``ceil(3 * sigma)``, so the kernel window size is ``2 * ceil(3 * sigma) + 1``.
```

Comment:
We've devided some time ago that we don't use 2nd person in our documentation.
```suggestion
Gaussian blur is calculated by applying a convolution with a Gaussian kernel, which can be
parameterized with ``windows_size`` and ``sigma``.
If only the sigma is specified, the radius of the Gaussian kernel defaults to
``ceil(3 * sigma)``, so the kernel window size is ``2 * ceil(3 * sigma) + 1``.
```

Reply:
Done